packages/radix-ui-themes/src/components/base-menu.css

```diff
@@ -30,17 +31,18 @@
   height: var(--base-menu-item-height);
   padding-left: var(--base-menu-item-padding-left);
   padding-right: var(--base-menu-item-padding-right);
-  position: relative;
   box-sizing: border-box;
+  position: relative;
   outline: none;
-  cursor: var(--cursor-menu-item);
   scroll-margin: var(--base-menu-content-padding) 0;
 
   /* Fix sticky text highlighting after selection in Firefox */
   user-select: none;
 
+  /* Cursors */
+  cursor: var(--cursor-menu-item);
   &:where([data-disabled]) {
-    pointer-events: none;
```

Comment:
- Why was there pointer-events: none?
- In theory, we are supposed to use disabled cursors here, but they don't feel right in menus